actix-server/src/handlers/deal_handler.rs

```diff
@@ -153,3 +158,180 @@ pub async fn update_deal(
     .await?;
     Ok(HttpResponse::Ok().json(deal))
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
+#[serde(rename_all = "snake_case")]
```

Comment:
```suggestion
```

I think this should be a different PR. Because if we're going to do it, we should do it to everything at once.